packages/sdk/src/tests/contracts/Holograph.test.ts

```diff
@@ -1,42 +1,209 @@
 import {beforeAll, describe, expect, it} from 'vitest'
 
```

Comment:
change this filename to follow the same structure that you defined for the contracts

from: packages/sdk/src/tests/contracts/Holograph.test.ts
to: packages/sdk/src/tests/contracts/holograph.contract.test.ts

the same for registry one